tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Models/GenericResponseVM.cs

```diff
@@ -10,5 +10,6 @@ public class GenericResponseVM
         public string Code { get; set; }
         public string Value { get; set;  }
         public bool IsError { get; set; }
+        public string Description { get; set; }

```

Comment:
Added extra property called Description which will be used when sending error information or any other information  to the client